bundles/audience_targeting.rst

```diff
@@ -58,64 +58,56 @@ And the routes for the website in the corresponding configuration file
 Finally the cache has to be correctly configured. You have the choice between
 the Symfony Cache and Varnish.
 
-For the `Symfony cache`_ the front controller for the website at
-`web/website.php` has to be told to make use of the user context feature. You
-do that by uncommenting the line with the `WebsiteCache` and adding `true` as a
-second parameter:
+For the `Symfony cache`_ the audience targeting cache listener needs to be added.
+This is possible in the constructor of the `WebsiteCache` in `app/WebsiteCache.php`:
 
 .. code-block:: php
 
-    <?php
-    $kernel = new WebsiteCache($kernel, true);
+    public function __construct(HttpKernelInterface $kernel, $cacheDir = null)
+    {
+        parent::__construct($kernel, $cacheDir);
+
+        $this->addSubscriber(new AudienceTargetingCacheListener());
+    }
 
 If you want to use the more powerful `Varnish`_ instead, you have to install it
-on your machine and configure it using a VCL. The following example can be used
-for that, however, it requires the header module, which comes with the
-`varnish-modules`_ package.
+on your machine and configure it using a VCL.
 
-.. code-block:: c
+The following will add full caching support inclusive audience targeting for Sulu:
 
+.. code-block:: varnish4
+
+    # /etc/varnish/default.vcl
     vcl 4.0;
 
-    import header;
+    include "<PATH-TO-SULU>/src/Sulu/Bundle/HttpCacheBundle/Resources/varnish/sulu.vcl";
+    include "<PATH-TO-SULU>/src/Sulu/Bundle/AudienceTargetingBundle/Resources/varnish/sulu.vcl";
```

Comment:
I think I would write `<PATH-TO-SULU-MINIMAL>/vendor/sulu/sulu` instead. I guess many people don't know that there is another package and where it is located.